### PR TITLE
add volunteer Pritam1136

### PR DESCRIPTION
**Pull request by Pritam1136**
I have added myself in data/users file.

### CODE REVIEW

**Comment by Pritam1136** [comment]
Hello @codewithvoid ,

Can you please merge this pull request.

File: data/users/pritam1136.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (Pritam1136) [inline data/users/pritam1136.json:1]: when this commit will merge?

Comment (codewithvoid) [reply data/users/pritam1136.json:1]: Hey apologies for the delay. Added some comments. Please check.

File: data/users/pritam1136.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "name": "Pritam Roy",
+    "bio": "C developer | Full-stack development ",
```
Comment (codewithvoid) [inline data/users/pritam1136.json:3]: Use 1 title.
```json
"bio": "Full Stack Developer"
```

File: data/users/pritam1136.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "image": "https://avatars.githubusercontent.com/u/126561455?s=400&u=dceb8af41ffd4061a38b03eb8d38c4f2e837d8ee&v=4",
```
Comment (codewithvoid) [inline data/users/pritam1136.json:4]: Use simpler image url
```json
"image": "https://avatars.githubusercontent.com/u/126561455?v=4"
```

File: data/users/pritam1136.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    "categories": [
+      {
+        "slug": "web developer",
```
Comment (codewithvoid) [inline data/users/pritam1136.json:7]: fix slug
```json
"slug": "web-development"
```

File: data/users/pritam1136.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+        "topics": ["ReactJS", "NodeJS","ExpressJS","TailwindCSS","git & github","Firebase"]
```
Comment (codewithvoid) [inline data/users/pritam1136.json:8]: pick max 2-3 items.

**Review by codewithvoid — CHANGES_REQUESTED** [review]

**Pull request closed without merging**